### DISCUSSION TITLE
2.6 实现类和对象

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#111] [Announcements]
# 2.6 实现类和对象

SICP Python 中文翻译

https://composingprograms.netlify.app/2/6

**Comment by sil3nceyang** [comment]
虽然学了CS50， python crash 书籍，还是晕倒在了2.6

**Comment by Alexababab** [comment]
人看嘛了😭

**Comment by haoaaa-111** [comment]
看完一段就复制给gpt让他提问 然后写自己的理解 让gpt判题 会发现自己啥都没看懂
